lib/utapi/utilities.js

```diff
@@ -49,7 +49,11 @@ function _listBucketMetrics(host,
             }
         });
     });
-    auth.client.generateV4Headers(request, '', accessKey, secretKey, 's3');
+    // TODO: cleanup with refactor of generateV4Headers
+    request.path = '/buckets';
+    auth.client.generateV4Headers(request, { Action: 'ListMetrics' },
+        accessKey, secretKey, 's3');
+    request.path = '/buckets?Action=ListMetrics';
```

Comment:
So, the problem was that `auth.client.generateV4Headers` expects `request.path` to not have any query params in it, but we cannot make a HTTP request to Utapi without those query params.
We would be refactoring the Arsenal method soon, so for now, we remove the query params before generating the headers and put them back for making the request.